workspace_tools/project.py

```diff
@@ -72,6 +84,24 @@
         print '\n'.join(map(str, sorted(TEST_MAP.values())))
         sys.exit()
 
+    # Only prints matrix of supported IDEs
+    if options.supported_ides:
+        print mcu_ide_matrix()
+        exit(0)
+
+    # Only prints matrix of supported IDEs
+    if options.supported_ides_html:
+        html = mcu_ide_matrix(verbose_html=True)
+        f = open("./export/README.md","w")
+        try:
+            f.write("Exporter IDE/Platform Support\n")
```

Comment:
You know you are writing HTML to git Markdown language file?
Two solutions:
1. Generate Git Markdown language and pop it inside READM.md inside exports or
2. Generate HTML and pop it into some HTML file (proper name and extension).

I propose you to export your HTML report data to file like ./export/EXPORTS.html.